.travis.yml

```diff
@@ -13,3 +13,6 @@ script:
   - python manage.py makemigrations
   - python manage.py migrate 
   - python manage.py test
```

Comment:
It would be nice if we used pytest instead to run the tests.
`python py.test`
This will also require that we add a pytest.ini configuration file
```
[pytest]
DJANGO_SETTINGS_MODULE=authors.settings
addopts = --cov=.

```

Reply:
@kalsmic we are now using coverage to run the tests, thanks for the idea